examples/specs/box_plot_minmax_full.vl.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A box plot showing median, min, and max in the US population distribution of age groups in 2000.",
+  "data": {"url": "data/population.json"},
+  "layer": [
+        {
+          "mark": "rule",
+          "encoding": {
+            "x": {"field": "age","type": "ordinal"},
+            "y": {
+              "aggregate": "min",
+              "field": "people",
+              "type": "quantitative",
+              "axis": {"title": "population"}
+            },
+            "y2": {
+              "aggregate": "max",
+              "field": "people",
+              "type": "quantitative"
+            },
+            "size": {"value": 5},
+            "color": {"value" : "white"}
```

Comment:
It depends, but more importantly --  why do you make plot white?  
(Basically, if things are white, they are not visible.)

We might want to introduce a mark definition syntax for parts of box marks. 

```
mark: {
  type: 'box',
  upperWhisker: {
     ... // MarkConfig 
  },
  lowerWhisker/whiskers/lowerBox/upperBox/boxes/middleTick/upperTick/lowerTick: ...
} 
```